test/powershell/Modules/Microsoft.PowerShell.Utility/clixml.tests.ps1

```diff
@@ -1,5 +1,9 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
+
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSAvoidUsingConvertToSecureStringWithPlainText', '')]
+param()
```

Comment:
Was added to just suppress the codefactor issue in 26daabad5. Happy to remove if we don't want to suppress.